server/src/main/java/com/adobe/testing/s3mock/FileStoreController.java

```diff
@@ -664,22 +662,35 @@ public CopyObjectResult copyObject(@PathVariable final String destinationBucket,
   @ResponseBody
   public CopyObjectResult copyObject(@PathVariable final String destinationBucket,
       @RequestHeader(value = COPY_SOURCE) final ObjectRef objectRef,
+      @RequestHeader(value = METADATA_DIRECTIVE,
+          defaultValue = "COPY") final String metadataDirective,
```

Comment:
use the constant `METADATA_DIRECTIVE_COPY` here?

Reply:
✅